(Newbie) problems following install instructions

Hi @mitsuhiko and @singingwolfboy thanks for all the work you do on lektor.

Disclaimer: I am new to lektor and also webpack.

When I follow the installation instructions I have two problems:

#### `package.json` not complete

    npm install --save-dev webpack babel-core node-sass babel-loader sass-loader css-loader url-loader style-loader file-loader extract-text-webpack-plugin 
    npm ERR! addLocal Could not install /home/oliver/work/oliver.bestwalter.de/lektor/webpack
    npm ERR! Linux 4.8.13-1-ARCH
    npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "--save-dev" "webpack" "babel-core" "node-sass" "babel-loader" "sass-loader" "css-loader" "url-loader" "style-loader" "file-loader" "extract-text-webpack-plugin"
    npm ERR! node v7.3.0
    npm ERR! npm  v4.0.5

    npm ERR! No name provided in package.json
    npm ERR! 
    npm ERR! If you need help, you may report this error at:
    npm ERR!     <https://github.com/npm/npm/issues>

    npm ERR! Please include the following file with any support request:
    npm ERR!     /home/oliver/work/oliver.bestwalter.de/lektor/npm-debug.log

Adding a name key to `package.json` then leads to the next error that a version is mising. If I add that also everything works fine. I am guessing that this changed at some point and the fields were made mandatory? The current [docs](https://docs.npmjs.com/getting-started/using-a-package.json#requirements) look that way.

So my guess would be that the README needs to be changed to something like this:

    {
      "name": "lektor-project-webpack",
      "version": "1.0.0",
      "private": true
    }

#### npm needs to be run inside webpack folder

As a newbie to lektor and webpack I naively expected everything to work if I run the commands from the lektor project, but the npm command needs to be run from inside the webpack folder.

--- a/README.md
+++ b/README.md
@@ -24,10 +24,12 @@ will need a `package.json` as well as a `webpack.config.js`
 ### `webpack/package.json`
 
 This file instructs `npm` which packages we will need.  All we need for a
-start is to create an almost empty file:
+start is to create an almost empty file (name and version fields are mandatory but not important for functionality, change them to suit your own needs):
 
 ```json
 {
+  "name": "lektor-webpack",
+  "version": "1.0.0",
   "private": true
 }
 ```
@@ -35,7 +37,8 @@ start is to create an almost empty file:
 Now we can npm install the rest:
 
 ```
-$ npm install --save-dev webpack babel-core node-sass babel-loader sass-loader css-loader url-loader style-loader file-loader extract-text-webpack-plugin
+$ cd </path/to/your/lektor/project>  # if you are not in there already
+$ npm install --prefix ./webpack --save-dev webpack babel-core node-sass babel-loader sass-loader css-loader url-loader style-loader file-loader extract-text-webpack-plugin
 ```
 
 This will install webpack itself together with babel and sass as well as

--- a/README.md
+++ b/README.md
@@ -18,13 +18,14 @@ lektor plugins add lektor-webpack-support
 
 ## Creating a Webpack Project
 
-Next you need to create a webpack project in the `webpack/` folder.  You
-will need a `package.json` as well as a `webpack.config.js`
+Next you need to create a webpack project. Create a `webpack/` folder and
+inside that folder create `package.json` and a `webpack.config.js`
 
 ### `webpack/package.json`
 
 This file instructs `npm` which packages we will need.  All we need for a
-start is to create an almost empty file (name and version fields are mandatory but not important for functionality, change them to suit your own needs):
+start is to create an almost empty file (name and version fields are mandatory
+but not important for functionality, change them to suit your own needs):
 
 ```json
 {
@@ -37,8 +38,8 @@ start is to create an almost empty file (name and version fields are mandatory b
 Now we can npm install the rest:
 
 ```
-$ cd </path/to/your/lektor/project>  # if you are not in there already
-$ npm install --prefix ./webpack --save-dev webpack babel-core node-sass babel-loader sass-loader css-loader url-loader style-loader file-loader extract-text-webpack-plugin
+$ cd </path/to/your/lektor/project>/webpack
+$ npm install --save-dev webpack babel-core node-sass babel-loader sass-loader css-loader url-loader style-loader file-loader extract-text-webpack-plugin
 ```
 
 This will install webpack itself together with babel and sass as well as

